docs/SRS/SRS.tex

```diff
@@ -722,7 +774,7 @@ \section{Open Issues}
 \section{Off-the-Shelf Solutions}
 \subsection{Ready-Made Products}
 \begin{itemize}
-    \item OpenID Connect offers an authentication protocol that can be used to implement a user-based authentication flow for the application.
+    \item Auth0 offers an authentication protocol that can be used to implement a user-based authentication flow for the application.
```

Comment:
Can you rephrase this to say

Auth0 is compliant with the OpenID Connect authentication protocol that can be used to implement a user-based authentication flow for the application.